OverwatchHighlights/HighlightTypes.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace OverwatchHighlights
+{
+    public enum HighlightType : uint
+    {
+        SHUTDOWN = 0x00000001,
+        SHARPSHOOTER = 0x00000002,
+        BLANK = 0x00000003,
+        LIFESAVER = 0x00000004
```

Comment:
I'd prefer that these not be uppercase, and be space-aligned to keep stylistic consistency with the rest of the codebase.